### DISCUSSION TITLE
Help with Github Pages

### DISCUSSION THREAD

**Discussion by Josephglz** [community/community#20543] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello! I have a little question.
How could I have more than 1 repository with github pages joined by a domain? That is, I currently have my domain: josephglz.me which is directed to josephglz.github.io (https://github.com/Josephglz/josephglz.github.io) however, I have other repositories that are my projects, which I would like keep on web but with a subdomain, for example:
The following repository: https://github.com/Josephglz/MyWeather and that it redirects from: myweather.josephglz.me
How could I achieve this?

Thanks!

**Comment by jfullstackdev** [comment, marked as answer]
you can always create a new Org, even without members, that will not follow the custom domain you've set, so you can separate things you want to

for example, `zerosolutions-1` org, it will have the GitHub Pages domain, `zerosolutions-1.github.io` as if it's another user separate from your personal account, 

for example, I have my Org

https://github.com/devz-throwable

that in GitHub Pages, is like this 

<https://devz-throwable.github.io/>

**Reply by Josephglz** [reply]
Oh, thanks a lot! thats awesome idea.
